templates/FVP.adapter.json

```
@@ -29,15 +29,24 @@
             }
         },
         "args": [
```

I.e., we can remove the dynamic shell selection per platform.
```suggestion
"args": [
```